python/paddle/fluid/tests/unittests/test_executor_and_mul.py

```diff
@@ -39,6 +39,49 @@ def test_mul(self):
         self.assertEqual((100, 100), out.shape)
         self.assertTrue(numpy.allclose(out, numpy.dot(a_np, b_np)))
 
+    def test_prepare_then_run(self):
+        a = data(name='a', shape=[784], dtype='float32')
+        b = data(
+            name='b',
+            shape=[784, 100],
+            dtype='float32',
+            append_batch_size=False)
+        c = data(
+            name='c', shape=[100, 10], dtype='float32', append_batch_size=False)
+        out = mul(x=a, y=b)
+        place = core.CPUPlace()
+        a_np = numpy.random.random((100, 784)).astype('float32')
+        b_np = numpy.random.random((784, 100)).astype('float32')
+        c_np = numpy.random.random((100, 10)).astype('float32')
+        exe = Executor(place)
+        feed = {'a': a_np, 'b': b_np, 'c': c_np}
+
+        prepared_ctx = exe.prepare(feed=feed, fetch_list=[out])
```

Comment:
do you plan to expose prepare as a public member?

Reply:
yes, prepare should be public, because a user should use it directly.